build/ci.sh

```diff
@@ -1,3 +1,9 @@
 set -e  # Exit immediately if a command exits with a non-zero status.
 
 npm run lint
+npm run test:cover
```

Comment:
`npm run test:cover` is already running the unit tests (so that it can actually generate coverage).